src/main/resources/assets/opencomputers/lang/de_DE.lang

```diff
@@ -183,17 +192,17 @@ oc:gui.Error.NoCPU=Im Computer ist kein Hauptprozessor (CPU) installiert.
 oc:gui.Error.NoEnergy=Nicht genug Energie.
 oc:gui.Error.NoRAM=Im Computer ist kein RAM installiert.
 oc:gui.Error.OutOfMemory=Nicht genug Arbeitsspeicher.
-oc:gui.Manual.Blocks=OpenComputers Blöcke
+oc:gui.Manual.Blocks=OpenComputers-Blöcke
 oc:gui.Manual.Home=Startseite
-oc:gui.Manual.Items=OpenComputers Gegenstände
+oc:gui.Manual.Items=OpenComputers-Gegenstände
 oc:gui.Manual.Warning.BlockMissing=Block nicht verfügbar.
 oc:gui.Manual.Warning.ImageMissing=Bild nicht gefunden.
 oc:gui.Manual.Warning.ItemMissing=Gegenstand nicht verfügbar.
 oc:gui.Manual.Warning.OreDictMissing=Ore-Dictionary-Eintrag nicht verfügbar.
-oc:gui.Raid.Warning=§4Eine Platte einzufügen löscht sie.[nl] Eine Platte zu entfernen löscht das Raid.
+oc:gui.Raid.Warning=§4Festplatten werden beim Einsetzen gelöscht.[nl] Beim Entfernen einer Platte wird das gesamte Raid gelöscht.
```

Comment:
Right, that text is displayed in the raid GUI, forgot about that. Even if there are space issues, Festplatte could be shortened to Platte, and the second part could be written as "Entfernen einer Platte löscht das Raid."